singer-io/tap-google-search-console
# Fix sync paging, bookmarking, attribution window

# Description of change
Fix/simplify bookmarking and paging issues for organizations with a large number of results. Add 14 day attribution window to account for results lag time.
Verified issue with paging/bookmarking by reducing page size limit from 25,000 to 1,000.
Changed paging to use 1,000 offset. Fixed issue with paging/body params not updating.
Verified bookmarking issue. Data results are not able to be sorted.
Changed bookmarking to bookmark after ALL pages of each site-sub_type.

# Manual QA steps
Ran Discovery, singer-check-tap, initial load sync, and incremental sync for all endpoints. Verified date windows with attribution window, verified multiple pages, and verified state write_bookmark messages.

# Risks
 Low risk. Clients may want to re-sync from start_date forward.
 
# Rollback steps
Revert branch to v.0.0.5
